src/content/en/showcase/2017/bear71.md

```diff
@@ -0,0 +1,50 @@
+{# wf_featured_image: /web/showcase/2017/images/bear71/featured.png #}
+{# wf_featured_snippet: Bear 71 is an interactive documentary launched by the National Film Board of Canada in 2012 to much critical acclaim. When it was built, Flash was in its golden years but with the deprecation of browser support for Flash, we were eager to explore WebVR, both as a way to preserve the project’s legacy and a chance to innovate how the story is experienced. #}
+{# wf_tags: webvr,casestudy #}
+
+# Bear 71 and WebVR {: .page-title }
+
+<img src="/web/showcase/2017/images/bear71/featured.png" class="attempt-right">
+
+### TL;DR {: .hide-from-toc }
+
+WebVR is an API built into browsers that combines stereo rendering with real-time head tracking, enabling a quick and easy way to enjoy VR content online. With WebVR, content creators can create immersive VR content that lives online and runs on a wide range of VR hardware.
+
+## About Bear 71
+
+Bear 71 is an interactive documentary produced by the National Film Board of Canada. Originally built in Flash, Bear 71 was released in 2011 to critical acclaim. The backbone of experience is a 23 minute track of audio and video, highlighting the relationship between humans, nature, technology, and one particular brown bear. The viewer is a voyeur in a world of information represented as an abstract grid of symbols. Surprisingly, this doesn't hamper the emotional story within, and it should be experienced first-hand to be understood.
+
+## Challenge
+
+The original Bear 71 was developed in Flash; arguably the best interactive storytelling medium of the time. A lot of care went into the technology and it was considered cutting edge. 5 Years later, the original vision holds up and story is still very relevant, but the technology behind it needs an update. Any technology, used creatively, can assist a good story, but Virtual Reality deserves special attention. VR has come and gone several times before, but it’s finally arrived in the mainstream. This is a huge opportunity for storytelling. Stories which traditionally happened in front of you, can now happen around you in VR.
+
+When Bear 71 was built, Adobe Flash was in its golden years. Each passing year, Flash's market share shrinks, and with that, previously accessible works are harder and harder to view. But with Flash's decline has come the maturation of HTML 5. Furthermore, VR is being taken more seriously than ever as a real platform for storytelling.
+
+To our eyes and ears, VR is naturally immersive. But VR has several barriers in its way. At first glance, the variety of headsets should suit many needs and budgets, and the variety of app stores involved in distribution is more choice. But users are known to prefer to do as much in a browser as possible. Users prefer to download and install as little as possible. For users and content creators alike, the appeal of HTML5 is the fact that rich experiences are just one URL away. This is where WebVR comes in.
```

Comment:
Combine first two sentences here: To our eyes and ears, VR is naturally immersive, but VR has several barriers in its way.